source/base/function_signed_distance.cc

```diff
@@ -166,6 +166,42 @@ namespace Functions
 
 
 
+    template <int dim>
+    Tensor<1, dim>
+    Ellipsoid<dim>::gradient(const Point<dim> & point,
+                             const unsigned int component) const
+    {
+      AssertIndexRange(component, this->n_components);
+      (void)component;
+
+      const Point<dim> point_in_centered_cosy =
```

Comment:
What does the term `cosy` refer to here?

Reply:
For me cosy is an abbreviation for coordinate system but it is probably not worth the possible confusion. I changed it and spelled it out.